GameWorld/HexGrid/DrawableHexSphere.cs

```diff
@@ -16,7 +16,8 @@ public class DrawableHexSphere
 
         private const int PerlinCoefficient = 80;
 
-        private float waterHeight = 0.5f;
+        private double waterHeight = 0.5f;
+        private int desiredWaterCoveragePercent = 99;
```

Comment:
Agree but it would be good to have several constructors - one having desiredWaterCoveragePercent and water level being calculated from that percent,
and one for having only absolute water level, in which case calculation of water level from desiredWaterCoveragePercent should be ignored.